[tutorial] update install commands for create astro (#11480)
withastro/docs: src/content/docs/en/tutorial/1-setup/2.mdx

      <Fragment slot="npm">
      ```shell
      # create a new project with npm

>       npm create astro@lastest

Reviewer: ```suggestion
      npm create astro@latest
```

      <Fragment slot="npm">
      ```shell
      # create a new project with npm
      npm create astro@latest
      ```
      </Fragment>
      <Fragment slot="pnpm">
      ```shell
      # create a new project with pnpm
      pnpm create astro@latest
      ```
      </Fragment>
      <Fragment slot="yarn">
      ```shell
      # create a new project with yarn
      yarn create astro
      ```
      </Fragment>
    </PackageManagerTabs>

> 2. Press `y` to install `create-astro`.

Reviewer 2: ```suggestion
2. Enter `y` to install `create-astro`.
```

> 2. Confirm `y` to install `create-astro`.

Reviewer: ```suggestion
2. Confirm with `y` to install `create-astro`.
```
Point Nr. 2 seems a bit odd to me, I think "Confirm with y to install create-astro" is better? Unless that's me not knowing English.

3. When the prompt asks you where to create the project, type in the name of a folder to create a new directory for your project, e.g.
`./tutorial`

    :::note
    A new Astro project can only be created in a completely empty folder, so choose a name for your folder that does not already exist!
    :::

4. You will see a short list of starter templates to choose from. Use the arrow keys (up and down) to navigate to the minimal (empty) template, and then press return (enter) to submit your choice.

> 5. When the prompt asks you whether or not to install dependencies, type or select `y`.

Reviewer 2: ```suggestion
5. When the prompt asks you whether or not to install dependencies, enter `y`.
```

> 6. When the prompt asks you whether or not to initialize a new git repository, type or select `y`.

Reviewer 2: ```suggestion
6. When the prompt asks you whether or not to initialize a new git repository, enter `y`.
```
